packages/core/src/codewhisperer/util/editorContext.ts

```
@@ -119,23 +120,55 @@ export async function buildListRecommendationRequest(
 
     logSupplementalContext(supplementalContexts)
 
+    // Get predictionSupplementalContext from PredictionTracker
+    let predictionSupplementalContext: codewhispererClient.SupplementalContext[] = []
+    try {
+        if (predictionTracker) {
+            predictionSupplementalContext = await predictionTracker.generatePredictionSupplementalContext()
+        }
+    } catch (error) {
+        getLogger().error(`Error getting prediction supplemental context: ${error}`)
```

You have the final call, but is it dangerous to swallow the error here? If a user is having issues we will never have any telemetry for this.

---

It's not dangerous, we are collecting initial data, and from the bug-bash no issues has been found. Telemetry will come later in this project. 
The main thing is if anything goes wrong it doesn't crash the inline completions flow, hence we swallow it here.